migration-engine/core/src/state.rs

```diff
@@ -56,14 +56,16 @@ impl EngineState {
         path: &str,
         f: ConnectorRequest<O>,
     ) -> CoreResult<O> {
+        let config_dir = std::path::Path::new(path).parent();
```

Comment:
would that work for relative paths? (e.g. `./schema.prisma`)

Reply:
This is basically just copied from your `db execute` PR :)

Reply:
hmm we should check that too